setup.py

```diff
@@ -47,6 +47,14 @@
         "git+https://github.com/genialis/resolwe.git@master#egg=resolwe-1.3.99",
     ),
     install_requires=(
+        # XXX: Need to manually specify these, otherwise
+        # 'python setup.py install --force' (used by Read the Docs) fails with:
+        # error: Django 1.10.1 is installed but Django<1.10 is required by {'django-haystack'}
+        'django>=1.9,<1.10a1',
+        # XXX: Need to manually specify these, otherwise
+        # 'python setup.py install --force' (used by Read the Docs) fails with:
+        # djangorestframework 3.4.7 is installed but djangorestframework==3.3.2 is required by {'resolwe'}
+        'djangorestframework==3.3.2',
```

Comment:
As soon as possible so that we have time to test it intensively by the next deploy at Rochester.